utils/installer/update_lvim.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$HOME/.local/share/lunarvim"}"
+LUNARVIM_CONFIG_DIR="${LUNARVIM_CONFIG_DIR:-"$HOME/.config/lvim"}"
+
+function msg() {
```

Comment:
Let's not implement the same function multiple times, create a new file and source it here

Reply:
It's a bit tricky to source it in the installer when it's being curl'd in 😅